contracts/shared/helpers/Fixture.ts

```diff
@@ -64,29 +59,35 @@ export default class Fixture {
       ).wait();
     } catch (e) {}
 
+    const bls = (await create2Fixture.create2Contract("BLSOpen")) as BLSOpen;
     // deploy Verification Gateway
-    const vgContract = await create2Fixture.create2Contract(
+    const verificationGateway = (await create2Fixture.create2Contract(
       "VerificationGateway",
-    );
-    const bls = (await create2Fixture.create2Contract("BLSOpen")) as BLSOpen;
-
-    try {
-      await (
-        await vgContract.initialize(bls.address, blsWalletImpl.address)
-      ).wait();
-    } catch (e) {}
+      ethers.utils.defaultAbiCoder.encode(
+        ["address", "address"],
+        [bls.address, blsWalletImpl.address],
+      ),
+    )) as VerificationGateway;
+
+    // try {
+    //   await (
+    //     await verificationGateway.initialize(bls.address, blsWalletImpl.address)
+    //   ).wait();
+    // } catch (e) {}
 
     // deploy BLSExpander Gateway
-    const blsExpander = await create2Fixture.create2Contract("BLSExpander");
-    try {
-      await (await blsExpander.initialize(vgContract.address)).wait();
-    } catch (e) {}
-
-    const verificationGateway = VerificationGateway__factory.connect(
-      vgContract.address,
-      vgContract.signer,
+    const blsExpander = await create2Fixture.create2Contract(
+      "BLSExpander",
+      ethers.utils.defaultAbiCoder.encode(
+        ["address"],
+        [verificationGateway.address],
+      ),
     );
 
+    // try {
+    //   await (await blsExpander.initialize(vgContract.address)).wait();
+    // } catch (e) {}
```

Comment:
Are these commented blocks still needed if new contracts are being deployed? Keeping them for reference?

Reply:
I'll get rid of those. It was was from when the gateway and expander were upgradable, but that is no longer the case.
Namely so that wallet holders needn't fear that the VG will get upgraded on them. The opt-in model is now the case where a wallet holder can change their VG.